### PR TITLE
Lb force fix

### PR DESCRIPTION
**Pull request by fweik**
Fixes #

Description of changes:
 - Was #1098 

PR Checklist
------------
 - [x] Tests?
   - [x] Interface
   - [x] Core 
 - [ ] Docs?

### CODE REVIEW

File: src/script_interface/lbboundaries/LBBoundary.hpp [hunk 1]
```diff
@@ -26,21 +25,28 @@ class LBBoundary : public AutoParameters {
...
+          [this](Variant const &value) {
+            m_lbboundary->set_charge_density(boost::get<double>(value));
+          },
+          [this]() { return m_lbboundary->charge_density(); },
+          VariantType::DOUBLE, 0},
+         {"net_charge",
+          [this](Variant const &value) {
+            m_lbboundary->set_net_charge(boost::get<double>(value));
+          },
+          [this]() { return m_lbboundary->net_charge(); }, VariantType::DOUBLE,
```
Comment (KaiSzuttor) [inline src/script_interface/lbboundaries/LBBoundary.hpp:39]: Can you please apply a nicer formatting for better readability?

Comment (fweik) [reply src/script_interface/lbboundaries/LBBoundary.hpp:39]: No, this is the correct formatting.

Comment (KaiSzuttor) [reply src/script_interface/lbboundaries/LBBoundary.hpp:39]: It seems to me that autoformatting is not always the most readable version.

**Comment by fweik** [comment]
Still beats discussing formatting and fiddling by hand...

File: testsuite/python/CMakeLists.txt [hunk 2]
```diff
@@ -16,7 +16,8 @@ set(py_tests  bondedInteractions.py
               particle.py
               rotational_inertia.py
               lbgpu_remove_total_momentum.py
-              tabulated.py
+              tabulated.py,
```
Comment (KaiSzuttor) [inline testsuite/python/CMakeLists.txt]: please remove comma, it's the reason for the failing testcase

**Review by hmenke — APPROVED** [review]

**Pull request merged**